website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -333,6 +333,10 @@ definition at provider level.
 can be used to lookup ID by name.
 * `edge_cluster_id` - (Optional) Specific Edge Cluster ID if required
 * `dedicate_external_network` - (Optional) Dedicating the external network will enable Route Advertisement for this Edge Gateway. Default `false`.
+* `non_distributed_routing_enabled` - (Optional, *v3.14+*) Non-Distributed routing will allow
```

Comment:
Looking at other occurrences, this would seem enough :)

```suggestion
* `non_distributed_routing_enabled` - (Optional, *v3.14+*) Non-distributed routing will allow
```

Reply:
Changed here and also in changelog.